index.html

```diff
@@ -280,28 +222,63 @@ <h4 class="frame-name">CSS</h4>
             <li class="tabs">
               <h4 class="tab-title">Framework</h4>
               <a href="#">
-
                 <img src="./image/right.svg" alt="right" class="right-arrow" />
-
-                <img src="./image/right.png" alt="right" class="right-arrow" />
-
               </a>
             </li>
           </ul>
           <ul class="pro-languages separator-bottom">
             <li class="tabs">
               <h4 class="tab-title">Skills</h4>
               <a href="#">
-
                 <img src="./image/right.svg" alt="right" class="right-arrow" />
-
-                <img src="./image/right.png" alt="right" class="right-arrow" />
-
               </a>
             </li>
           </ul>
         </div>
       </div>
     </section>
+    <section class="form">
+      <ul class="form-header">
+        <li class="form-list">
+          <h4 class="contact-title">Contact me</h4>
+          <p class="contact-description">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </li>
+      </ul>
+      <form action="https://formspree.io/f/mgebvkpz" method="POST"  class="form-details">
+        <input
+        name="name"
+          class="username"
+          type="text"
+          id="name"
+          placeholder="Yeremias"
+          max-length="30"
+          required
+        />
```

Comment:
- [ ] Great job working on your contact section! 💯 However, there are some typos that you need to fix in order to make the validations work. In this case, the correct attribute for the length limit is `maxlength` and you have `max-length`, please consider changing it to make the validation work! 🤓 🔧 

```suggestion
        <input
        name="name"
          class="username"
          type="text"
          id="name"
          placeholder="Yeremias"
          maxlength="30"
          required
        />
```